sphinx/source/docs/user_guide/examples/graph_node_and_edge_attributes.py

```diff
@@ -0,0 +1,38 @@
+import networkx as nx
+
+from bokeh.io import show, output_file
+from bokeh.models import Plot, Range1d, MultiLine, Circle, HoverTool, BoxZoomTool, ResetTool
+from bokeh.models.graphs import from_networkx
+from bokeh.palettes import Spectral4
+
+# Prepare Data
+G = nx.karate_club_graph()
+
+SAME_CLUB_COLOR, DIFFERENT_CLUB_COLOR = "black", "red"
+edge_attrs = {}
+
+for start_node, end_node, _ in G.edges(data=True):
+    edge_color = SAME_CLUB_COLOR if G.nodes[start_node]["club"] == G.nodes[end_node]["club"] else DIFFERENT_CLUB_COLOR
+    edge_attrs[(start_node, end_node)] = edge_color
+
+nx.set_edge_attributes(G, edge_attrs, "edge_color")
+
+print("G has node attributes. For example, {}".format(G.nodes[0]))
+print("G has edge attributes. For example, {}".format(G.edges[(0, 1)]))
```

Comment:
This will fail in python 2, but also I don't think we generally print any output at all in any of our examples. I would suggest just removing these entirely to be consistent.

Reply:
Thank you for your pointed out. I fixed it.